### PR TITLE
INTMDB-454: New Data Source to GET Org Id

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

New Data Source to GET Org Id

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: website/docs/r/project.html.markdown [hunk 1]
```diff
@@ -15,9 +15,12 @@ description: |-
 ## Example Usage
 
 ```terraform
+data "mongodbatlas_roles_org_id" "test" {
+}
```
Comment (Zuhairahmed) [inline website/docs/r/project.html.markdown:19]: @martinstibbe should this data block be empty? Just confirming that no parameters are required to use this data source

**Review by Zuhairahmed — APPROVED** [review]
Added few formatting / link fixes. Otherwise LGTM. Thanks @martinstibbe!

File: website/docs/r/project.html.markdown (hunk 1 above)
Comment (martinstibbe) [reply website/docs/r/project.html.markdown:19]: No parameters just using API key to connect terraform to API gets org_id value

**Review by andreaangiolillo — APPROVED** [review]
[q] Why do we need this data source?

LGTM

**Comment by martinstibbe** [comment]
> [q] Why do we need this data source?
> 
> LGTM

Customer requests brought this feature request to find org id linked to API key

**Pull request merged**